omod/src/main/java/org/openmrs/module/patientflag_rest/extension/html/AdminList.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.patientflag_rest.extension.html;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.openmrs.module.Extension;
+import org.openmrs.module.web.extension.AdministrationSectionExt;
+
+/**
+ * This class defines the links that will appear on the administration page under the
+ * "patientflag_rest.title" heading. 
+ */
+public class AdminList extends AdministrationSectionExt {
```

Comment:
This probably isn't necessary since we already have an `AdministrationSectionExt` [here](https://github.com/openmrs/openmrs-module-patientflags/blob/master/omod/src/main/java/org/openmrs/module/patientflags/extension/html/FlagAdminExt.java), right?